src/templates/partials/lighttpd.hbs

```diff
@@ -11,36 +17,87 @@ $SERVER["socket"] == ":80" {
 {{/if}}
 }
 
+$HTTP["scheme"] == "https" {
+    # HTTP Strict Transport Security ({{output.hstsMaxAge}} seconds)
+    setenv.add-response-header = (
+        "Strict-Transport-Security" => "max-age={{output.hstsMaxAge}}"
+    )
+}
 {{/if}}
+
+{{#if (minver "1.4.56" form.serverVersion)}}
+# select one TLS module: "mod_openssl" "mod_mbedtls" "mod_gnutls" "mod_wolfssl" "mod_nss"
+#server.modules += ("mod_openssl")
+
+# lighttpd 1.4.56 and later will inherit ssl.* from the global scope if
+# $SERVER["socket"] contains ssl.engine = "enable" and no other ssl.* options
+# (to avoid having to repeat ssl.* directives in both ":443" and "[::]:443")
+$SERVER["socket"] ==     ":443" { ssl.engine = "enable" }
+$SERVER["socket"] == "[::]:443" { ssl.engine = "enable" }
+ssl.privkey = "/path/to/private_key"
+ssl.pemfile = "/path/to/signed_cert_plus_intermediates"
```

Comment:
Now that the pemfile contains intermediates, what would you think about clarifying the order with something like this
```suggestion
ssl.pemfile = "/path/to/signed_cert_followed_by_intermediates"
```

Reply:
ok.  ackn

Reply:
PR updated